src/server_manager/install_scripts/install_server.sh

```diff
@@ -388,6 +388,21 @@ Make sure to open the following ports on your firewall, router or cloud provider
 "
 }
 
+function set_hostname() {
+  # These are URLs that return the client's apparent IP address.
+  # We have more than one to try in case one starts failing
+  # (e.g. https://github.com/Jigsaw-Code/outline-server/issues/776).
+  local -ar urls=(
+    'https://ipinfo.io/ip'
```

Comment:
Perhaps we should prefer the Google service. I wonder if it will work on Oracle though.
We can also consider adding the dig approach.

Reply:
`ipinfo.io` has largely been working OK for us, so this seems like the most conservative change.